### ISSUE TITLE
Add a `Full` vote display mode? And what should be the default?

### ISSUE THREAD

**Issue by dessalines** [LemmyNet/jerboa#1400]
### Pre-Flight checklist

- [X] Did you check to see if this issue already exists?
- [X] This is a single feature request. (Do not put multiple feature requests in one issue)
- [X] This is not a question or discussion. (Use https://lemmy.ml/c/jerboa for that)

### Describe The Feature Request Below

I have a [PR to add these vote display modes to the lemmy back-end](https://github.com/LemmyNet/lemmy/pull/4450).

But we have two unresolved questions: 

1. Should we add a `Full` mode alongside `Score+Downvote` ?
2. What should be the default until the back end gets merged + deployed? 

For 1, I don't think its necessary, mainly because I'm trying to avoid this mess: ![Screenshot_2024-02-12-12-52-32-271-edit_com jerboa_1](https://github.com/dessalines/jerboa/assets/930215/c6793f47-0a47-4a95-8421-9002a2a3116e) . 

I removed all this distracting extra unecessary info, and put everything in the node header bar. But should it also show the upvotes alongside score? IE `score, upvotes, downvotes, time` . IMO that's too much info, and it isn't necessary.

For 2, I think we should switch the default to `Score+UpvotePct`. It looks the cleanest, and seems to be what most lemmy front-ends are using now.

For reference, here's what these modes look like in jerboa (without full)

Score + Downvote (the current default): 
![Screenshot_2024-02-14-17-58-41-653-edit_com jerboa debug](https://github.com/dessalines/jerboa/assets/930215/5168953a-e0f2-4b62-bc6d-c172bb874807)

Score and Upvote Pct (I'd like this to be the new default): 

![Screenshot_2024-02-14-17-57-01-796-edit_com jerboa debug](https://github.com/dessalines/jerboa/assets/930215/3f630aba-806e-41c5-a8e2-318f2611c645)

Upvote pct: 

![Screenshot_2024-02-14-17-57-39-362-edit_com jerboa debug](https://github.com/dessalines/jerboa/assets/930215/3f4d432d-24b9-46b6-9a6c-8c7159f1ac0f)

Upvote pct in list view: 

![Screenshot_2024-02-14-17-57-52-874-edit_com jerboa debug](https://github.com/dessalines/jerboa/assets/930215/0faac34e-619d-42dc-ae5c-acdd2fbb3828)

Hide all:

![Screenshot_2024-02-14-18-12-40-282-edit_com jerboa debug](https://github.com/dessalines/jerboa/assets/930215/d7182441-c485-4d35-9baf-78168e0fab39)

cc @MV-GH

**Comment by MV-GH** [comment]
Yeah I agree with the default change.

But some users do like seeing the upvotes near the arrows. Even tho, it is more visual clutter. And we can add this option easily since all the stuff is still there for it.

So the question is then, do make that Full and remove the Score + downvotes (Do users even want this option)?

Or do we add ScoreDownvotes option to the backend for this?

**Comment by dessalines** [comment]
> But some users do like seeing the upvotes near the arrows. Even tho, it is more visual clutter. And we can add this option easily since all the stuff is still there for it.

I actually did remove the score as a function param to post and comment footer/action bars. Now it only injects your `myVote`, to color the button. I did that so that item scores can all be shown in one spot, rather than 3 places.

> So the question is then, do make that Full and remove the Score + downvotes (Do users even want this option)?

I think the more options the better, and that one is already done, so there's no reason to remove it. I'll just change the name to `ScoreAndDownvote`, and also add this in my back-end PR.

And I'll add a `Full` option in a PR (with a screenshot, but it won't be in use until the back-end stuff is done).
